### PR TITLE
README: reference a local checkout of rules_haskell

### PR DESCRIPTION
**Pull request by Profpatsch**
Fixes https://github.com/tweag/rules_haskell/issues/494

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,8 +135,34 @@ See [rules_haskell_examples][] for examples of using these rules.
...
+
+```
+http_archive(
+    name = "io_tweag_rules_haskell",
+    strip_prefix = "rules_haskell-" + version,
+    sha256 = …,
+    urls = …,
+)
+```
+
```
Comment (guibou) [inline README.md]: Why not using the bazel flag `override_repository`?

```
--override_repository \
   io_tweag_rules_haskell=/path/to/local/checkout
```

Comment (Profpatsch) [reply README.md]: Oh, that might be even better on occasions. I’ll add a reference to that as well.

Comment (Profpatsch) [reply README.md]: We should write a blog post about the most helpful bazel command line flags …

Comment (guibou) [reply README.md]: Add the ref and I approve.

Comment (Profpatsch) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -135,8 +135,47 @@ See [rules_haskell_examples][] for examples of using these rules.
 
 [rules_haskell_examples]: https://github.com/tweag/rules_haskell_examples
 
+
```
Comment (mboes) [inline README.md]: Spurious whitespace change.

File: README.md [hunk 3]
```diff
@@ -135,8 +135,47 @@ See [rules_haskell_examples][] for examples of using these rules.
...
+1) Use the [`--override_repository`][override_repository]
+   command line option:
+   
+   ```
+   bazel build/test/run/sync \
+     --override_repository io_tweag_rules_haskell=/path/to/checkout
+   ```
+   
+2) Edit your `WORKSPACE` to use [`local_repository`][local_repository]
```
Comment (mboes) [inline README.md]: Remove this option. This is the wrong way to do it.

Comment (mboes) [reply README.md]: But bazelrc is even more right. It's the only mechanism that allows you to override on a per user basis without fearing committing the dirty changes in the WORKSPACE by mistake. See https://docs.bazel.build/versions/master/best-practices.html#bazelrc.

I'm not a fan of overlong README's. The bar should be high as to what content goes there. Let's avoid repeating the documentation there for things are bad practices anyways.

Comment (Profpatsch) [reply README.md]: Okay, replaced it with “If you don’t want to type that every time, temporarily add it to
   if you want a more permanent setup (just don’t commit it):	`.bazelrc`.”

Comment (mboes) [reply README.md]: Add the link I pointed you to.

Comment (Profpatsch) [reply README.md]: Oh, right. Done.

**Pull request merged**